application/src/main/java/com/softserve/lv460/application/dto/page/PaginationRequest.java

```diff
@@ -0,0 +1,23 @@
+package com.softserve.lv460.application.dto.page;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+
+@Getter
+@Setter
+public class PaginationRequest {
+  private Integer size;
+  private Integer page;
+  private SortRequest sort;
+
+  public PageRequest mapToPageRequest() {
+    PageRequest pageRequest = null;
+    if (sort == null) {
+      return PageRequest.of(page, size);
+    } else {
```

Comment:
you can omit "else" block and have return PageRequest.of(page, size, Sort.by(sort.getDirection(), sort.getFieldName()))